docs/migrate-matomo-3-to-4.md

```diff
@@ -0,0 +1,79 @@
+---
+category: Develop
+---
+# Migrate Plugin from Matomo 3.X to Matomo 4
+
+This migration guide covers explains how to do some migrations to make a plugin compatible with Matomo 4. A list of
+all changes in Matomo 4 can be found in the [Changelog](/changelog).
+
+## Adjust the required Matomo version
+
+For your plugin to be executed in Matomo 4 you first need to configure it as compatible with Matomo 4:
```

Comment:
would reword this from "configure it as compatibile" to "show it is compatible with matomo 4 in your plugin.json file"

Reply:
👍 changed